tests/test_covarites_assisted_embed.py

```diff
@@ -0,0 +1,303 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Nov 17 20:13:45 2019
+
+@author: jerryyao
+"""
+
+import unittest
+import graspy as gs
+import numpy as np
+from graspy.embed.casc import CovariateAssistedSpectralEmbed
+from graspy.embed.lse import LaplacianSpectralEmbed
+from graspy.embed.svd import selectSVD
+from graspy.simulations.simulations import er_np, er_nm, sbm
+from sklearn.cluster import KMeans
+from sklearn.metrics import adjusted_rand_score as ARI
+class TestStringMethods(unittest.TestCase):
+
+    def _testcasc_cca(self, *args, **kwargs):
+        A=[[0., 0., 1., 1., 1., 1., 1., 1., 1., 1., 0., 0., 1., 0., 0., 0.,

```

Comment:
I don't know where you got this matrix. maybe try to read it in from a file instead of copy paste it

Reply:
This is a specially generated matrix with significant difference between probability.I can put it in a file and read this from a file.But Where should I put that file?